enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  DEFAULT_EMBED_DISPLAY_OPTIONS,
+  useDashboardFullscreen,
+  useDashboardRefreshPeriod,
+  useEmbedTheme,
+  useRefreshDashboard,
+} from "metabase/dashboard/hooks";
+import type {
+  EmbedDisplayParams,
+  RefreshPeriod,
+} from "metabase/dashboard/types";
+import { PublicDashboard } from "metabase/public/containers/PublicDashboard/PublicDashboard";
+import { Box } from "metabase/ui";
+import type { DashboardId, ParameterId } from "metabase-types/api";
+
+export const StaticDashboard = ({
+  dashboardId,
+  parameterValues,
+  refreshPeriod: initialRefreshPeriod = null,
+  displayOptions = {},
+}: {
+  dashboardId: DashboardId;
```

Comment:
Let me know what you think of the API. I'm thinking we might want to get rid of the extra `displayOptions` and possibly remove `refreshPeriod`

Reply:
I think some of the `displayOptions` might not be applicable? For example:

- We already have `font` on the SDK config, so I guess this acts as an override
- Do we want to expose the `DisplayTheme` to allow night mode? I think this is a question we'll want to ask @albertoperdomo
- The other options `bordered`, `titled`, `hide*` sounds pretty useful to me, but maybe just add a docstring to document them a bit, and re-export this to SDK entry point so we can `import type { EmbedDisplayOptions } from `@metabase/react-embedding-sdk`

```ts
export type EmbedDisplayParams = {
  bordered: EmbedBordered;
  titled: EmbedTitle;
  hideDownloadButton: EmbedHideDownloadButton;
  hideParameters: EmbedHideParameters;
  font: EmbedFont;
  theme: DisplayTheme;
};
```

As for refreshPeriod, I think that sounds pretty useful to me, but worth a docstring as well. I think `dashboardId` and `parameterValues` is good as-is, just add a little docstring on what the SDK user should pass to `parameterValues`.